src/components/shared/BankData.vue

```diff
@@ -10,6 +10,10 @@
 		</li>
 		<li><strong>{{ $t( 'bank_data_bic')  }}:</strong> {{ $t( 'bank_data_operator_account_bic')  }}</li>
 		<li>{{ $t( 'bank_data_operator_bank_name')  }}</li>
+		<li v-if="bankTransferAmount">
+			<strong>{{ $t( 'bank_data_transfer_amount_label' ) }}: </strong>
+			<span class="bank-transfer-code"> {{ bankTransferAmount }}</span>
+		</li>
 		<li v-if="bankTransferCode">
 			<strong>{{ $t( 'bank_data_transfer_code' ) }}:</strong>
```

Comment:
One last tiny thing, there's no space between the bold text and the bank transfer code so lets add one in the strong tag:

```suggestion
			<strong>{{ $t( 'bank_data_transfer_code' ) }}: </strong>
```